service/src/main/resources/config/broad/cms_synpuf/expanded/cms_synpuf.json

```diff
@@ -16,5 +16,13 @@
+  "plugins": {
+    "TEST": {
+      "type": "bio.terra.tanagra.plugin.TestPluginExternalImplementation",
+      "parameters": {
+        "test": "configured value"
+      }
+    }
+  }
```

Comment:
I started trying to create a plugin-specific test underlay, but decided against it as doing so rapidly expanded into having to create a dozen additional supporting json files in order to the underlay to be valid enough to be loaded, which seemed unnecessary and difficult to maintain.

In the example you linked, the entities, entitygroups, and sql would also need to be duplicated into the test underlay in order for it to be valid. Is that what the ask is?

Reply:
Before I put it in the synpuf underlay I tried creating my own underlay with just the underlay file and a single entity and it failed multiple times, first because there were not data pointers defined, then because there were no attributes defined. At that point I figured I was going to have to fully build out a nonsense underlay in order to get it to build, so I stuck it in synpuf. Maybe I missed something, I'll try again on Monday.